docs/features/column-level-lineage.md

```diff
@@ -34,6 +38,69 @@ The transformation type is also displayed for each column, which will help you u
 | Unknown | We have no information about the transformation type. This could be due to a parse error, or other unknown reason. | 
 
 
-## Limitation
+## Impact Radius of a Column
+
+The **right side of the Column-Level Lineage (CLL)** graph represents the **impact radius** of a selected column.
+This view helps you quickly understand what will be affected if that column changes.
+
+### What does the impact radius include?
+
+- **Downstream columns** that directly reference the selected column
+- **Downstream models** that directly depend on the selected column
+- **All indirect downstream columns and models** that transitively depend on it
+
+This helps you evaluate both the direct and downstream effects of a column change, making it easier to understand its overall impact.
+
+
+### Example: Simplified Model Chain
+
+Here are some example models
+
+```sql
+-- stg_orders.sql
+select
+  order_id,
+  customer_id,
+  status,
+  ...
+from {{ ref("raw_orders") }}
+
+
+-- orders.sql
+select
+  order_id,
+  customer_id,
+  status,
+  ...
+from {{ ref("stg_orders") }}
+
+
+-- customers.sql
+select
+  c.customer_id,
+  ...
+from {{ ref("stg_customers") }} as c
+join {{ ref("stg_orders") }} as o
+  on c.customer_id = o.customer_id
+where o.status = 'completed'
+group by c.customer_id
+
+
+-- customer_segments.sql
+select
+  customer_id,
+  ...
+from {{ ref("customers") }}
+```
+
+![alt text](../assets/images/features/cll-example.png){: .shadow}
+
+Here's how changes to `stg_orders.status` affect downstream models:
```

Comment:
Remove this sentence, as I moved it to the beginning of this section:

> "Here's how changes to `stg_orders.status` affect downstream models:"

Replace with:

> "The following impact is detected:"

Reply:
done